bcda/auth/plugin/alpha_test.go

```diff
@@ -145,8 +145,13 @@ func (s *AlphaAuthPluginTestSuite) TestRevokeAccessToken() {
 }
 
 func (s *AlphaAuthPluginTestSuite) TestValidateAccessToken() {
-	err := s.p.ValidateAccessToken("")
-	assert.Equal(s.T(), "not yet implemented", err.Error())
+	userID := uuid.NewRandom().String()
+	acoID := uuid.NewRandom().String()
+	ts, _ := auth.InitAuthBackend().GenerateTokenString(userID, acoID)
+	err := s.p.ValidateAccessToken(ts)
+	assert.Nil(s.T(), err)
+	err = s.p.ValidateAccessToken("this.token.fails")
```

Comment:
Instead of calling it CustomClaims, can we rename it to AllClaims, or something along those lines? It does contain all of them, not just the custom claims ;)

Reply:
I thought `CustomClaims` was a good name.  I know it's not _just_ custom claims.  but it is our custom set of claims including some standard and some custom claims.  I changed it to `AllClaims` anyway but feel free to change again in a later PR if you think it could be clearer